CUP/CSAT/etc. BRH, Basic ammo greenmag compat, MMG Tracers, [Inactive] Ammo config

Alien314 opened the pull request:
Applies BRH to CSAT, CUP, and some other vanilla uniforms that previously had very little ballistic protection.
Removed cuz it just broke RHSUSF vests and CUP iotv has the same armor = 20 line as RHSUSF, not sure what's up with the "5*0" on vanilla stuff... _Removes possible graze protection from RHSUSF vests, I think, CUP at least applies this universally but is overwritten for RHSUSF._
Greenmag Basic ammo compat for most if not all CUP/RHS 10+ round magazines.
Adds red, yellow, and green tracer belts for Vanilla MMGs.
[Inactive] Full ammo config, mostly balancing RHS around Vanilla and ACE values, commented until it matters /shrug.

Splits configs into separate files for readability/github diffs.

jdcaperon commented on addons/configtweaks/config.cpp:
```diff
@@ -174,14 +183,13 @@ class CfgVehicles {
-        depends =
-            "(((Total - 0.25) max 0) + ((HitHead - 0.25) max 0)"
-            "+ ((HitBody - 0.25) max 0)) * 2";
+	depends="""(((Total - 0.25) max 0) + ((HitHead - 0.25) max 0)""" \n """+ ((HitBody - 0.25) max 0)) * 2""";
```
Up to you how you want to format this, but I'd say that this isn't the ideal state 😂

Alien314 replied:
It's how it was originally, at least in the old configTweaks and I guess BRH.

Alien314 replied:
Fixed to actual original BRH line now

jdcaperon commented on addons/configtweaks/config.cpp:
```diff
@@ -239,24 +247,426 @@ class CfgWeapons {
+	};
+};
+
+// Ammo Balance, normalizes 12G slugs across mods, example of what can be done with this
```
Worth noting that if a mod _does_ set its own values, the inheritance is overwritten, which is a pain to track down but ye.

Alien314 replied:
There aren't so many ammo calibers so it's not too hard to go through, and I found a spreadsheet with values for different mods, that at least makes it easier to know where to look.

jdcaperon reviewed (approved):
LGTM

jdcaperon reviewed (approved):
o7

Alien314 commented:
Ready for update now I think <3

jdcaperon commented:
@Alien314 I can probably push out a steam update for this unless y'all are using a local version now

Alien314 commented:
Still using your steam workshop version. <3  But yeah I think I've fixed it up well enough now.

Alien314 commented:
@jdcaperon I was thinking about figuring out how to put my fork up on steam workshop, if that's cool? I'll probably end up tweaking it a bunch as issues crop up.

jdcaperon commented:
> @jdcaperon I was thinking about figuring out how to put my fork up on steam workshop, if that's cool? I'll probably end up tweaking it a bunch as issues crop up.

Go for it!

Alien314 committed:
3DEN Array UI and layers

Alien314 committed:
reduce brightness

Alien314 committed:
fix and layer increment

MomoTempo committed:
Add M240 and Remove HK121 and LWMMG from MMG

Adding the M240B and M240G to the arsenal as options for MMG.

Removing the HK121 and LWMMG from MMG.

Created _itemWeaponSFMMG to allow the use of HK121 and LWMMG with SF units.

MomoTempo committed:
Updated MAAWS for MAT and SF

MAAWS updated for MAT and SF AT Operator.

Laser Range finding MAAWS is an SF weapon.

Conventional MAAWS is a MAT weapon.

Alien314 committed:
Add M240 and Remove HK121 and LWMMG from MMG



Adding the M240B and M240G to the arsenal as options for MMG.

Removing the HK121 and LWMMG from MMG.

Created _itemWeaponSFMMG to allow the use of HK121 and LWMMG with SF units.

Fixed _itemWeaponSFMMG being commented out.

MAAWS updated for MAT and SF AT Operator.

Laser Range finding MAAWS is an SF weapon.

Conventional MAAWS is a MAT weapon.

Alien314 committed:
replace rangefinder maaws in mat

Alien314 committed:
green WP nvg default

MomoTempo committed:
Adding 82mm Rangecard and Artillery Range Table to Equipment

Just giving the 82mm Range Card and Artillery Table to everyone in case we need it for some reason. Does prevent us having to worry about this in the future and helps mission makers just not have to worry about it as well.

Alien314 committed:
Adding 82mm Rangecard and Artillery Range Table to Equipment

Just giving the 82mm Range Card and Artillery Table to everyone in case we need it for some reason. Does prevent us having to worry about this in the future and helps mission makers just not have to worry about it as well.

Alien314 committed:
crew arty tables, medic 148

Alien314 committed:
unstableWake debug

MomoTempo committed:
Camos, Different Mag Colors, Kitbags sized items for ARs.

Various Camos for some rifles already in our arsenal.

At least two different mag colors for 5.56 and 7.62x51 SCAR rifles.

Kitbags for ARs.

Added more 28 lb load bags.

Alien314 committed:
clarify bleed setting

Alien314 committed:
remove collapse damage (for now)

Alien314 committed:
fix role/arsenal modules

Alien314 committed:
fix zeus uncon, and non-breaking errors

Alien314 committed:
Camos, Different Mag Colors, Kitbags sized items for ARs.

Various Camos for some rifles already in our arsenal.

At least two different mag colors for 5.56 and 7.62x51 SCAR rifles.

Kitbags for ARs.

Added more 28 lb load bags to _itemPackMedLight.

Added the new MCC weapon to rifles. It's basically the same but whatever.

Alien314 committed:
move greenmag arsenal button def

Alien314 committed:
preInit greenmag button for editor

Alien314 committed:
fix no pulse ignoring everything else

Alien314 committed:
remove red cross armbands

Alien314 committed:
MCC sights tweaks

MomoTempo committed:
MCC Sights Tweaks

Fixing the zeroing for the various Modular Combat Carbines (MCC) sights.

Any "red dot" type sight is set to 100m.

Magnified versions of those "red dots" when the magnifier is up is set/locked to 300m.

MomoTempo committed:
Fixing the indenting

Fixed an indentation.

MomoTempo committed:
Adding Various Additional Uniforms

Various additional CUP uniforms that I don't think appear on our typical Opfor.

Alien314 committed:
Adding Various Additional Uniforms

Various additional CUP uniforms that I don't think appear on our typical Opfor.

Alien314 committed:
MCC config is inconsistent

Alien314 committed:
MCC Sights Tweaks

Fixing the zeroing for the various Modular Combat Carbines (MCC) sights.

Any "red dot" type sight is set to 100m.

Magnified versions of those "red dots" when the magnifier is up is set/locked to 300m.

Alien314 committed:
CUP helo mass tweak

Alien314 committed:
Reduced damage and AoE for collapse

Alien314 committed:
ace took away obj burn, also fuel cans can burn now

Alien314 committed:
wake broke setUncon, softCap, and fainting

Alien314 committed:
fix house destruction, damage, CUP helo mass settings

Alien314 committed:
helo mass needs restart

Alien314 committed:
aar>ar temp, remove engi button

Alien314 committed:
get and setFuel

Alien314 committed:
scream

Alien314 committed:
simpleScale and improved stabToggle

MomoTempo committed:
Color Variants for Rifles and Carbines

mk17 CQC color variants added.

G3A3 color variants added.

ACR color variants added.

CZ Bren color variants added.

M4A1 SOPMOD Block II color variants added.

famas color variants added.

Alien314 committed:
Color Variants for Rifles and Carbines

mk17 CQC color variants added.

G3A3 color variants added.

ACR color variants added.

CZ Bren color variants added.

M4A1 SOPMOD Block II color variants added.

famas color variants added.

Alien314 committed:
Fix allowJam setting

Alien314 committed:
fix medic 148 access

Alien314 committed:
improved stab and helo stab

Alien314 committed:
vic flag, afk toggle and carry/drag shenanigans

Alien314 committed:
improved stab toggle, locke state global

Alien314 committed:
seb table modules, head tap

Alien314 committed:
set unit role adds previous nvg/glasses

Alien314 committed:
improvements and avionics damage because Idk lol

Alien314 committed:
head reduce

Alien314 committed:
improve simpleScale

Alien314 committed:
pat range increase

Alien314 committed:
Mortar  Team roles

Alien314 committed:
Table module improvements

MomoTempo committed:
Added Another Vest and Breaching Items for Leaders and Engineers

Added a CUP vest that doesn't look very rough.

Added breaching items for Leaders and Engineers.

MomoTempo committed:
Fixing breaching thingies

fixing oofies in the breaching area

MomoTempo committed:
Adding all MCC Reflex Sights

Including all the MCC reflex sights variants.

MomoTempo committed:
Adding remaining 3x MCC for full accountability

Adding remaining 3x MCC optics.

Alien314 committed:
vis laser nerf

Alien314 committed:
isMP check for sebTable (RIP)

Alien314 committed:
Boost headpat distance/headpat from above

Alien314 committed:
coPilot use vanilla hash, fix killed

Alien314 committed:
stuck attach contingency

Alien314 committed:
prevent drop carried/settings

Alien314 committed:
unstable wake post process effect

Alien314 committed:
Added Another Vest and Breaching Items for Leaders and Engineers

Added a CUP vest that doesn't look rough.

Added breaching items for Leaders, SF Leaders, and Engineers.

Also added MCC Reflex sights per end-user request.

Alien314 committed:
balance CPC vests

Alien314 committed:
pp settings

Alien314 committed:
backpack disposable "mags"

Alien314 committed:
fix backpack disposable inconsistency

Alien314 committed:
duplicate plate mass safeguards

Alien314 committed:
exit with message when unit in cardiac arrest

Alien314 committed:
allow loading conscious

Alien314 committed:
detach contingency, fix drop args, remEx doAnim

Alien314 committed:
impact smonk

Alien314 committed:
ar/aar kitbag

Alien314 committed:
non-player carry

Alien314 committed:
increase interval, text length/sort

Alien314 committed:
fix inconsistent class name

Alien314 committed:
improve garage

MomoTempo committed:
Adding more default camos to the arsenal

adding the woodland and various alternative camos for default NATO clothes to the arsenal for Jubi

Alien314 committed:
hide the rediculous

Alien314 committed:
disable handleUncon override, doAnim local to wake

MomoTempo committed:
adding new MCC M4A1 MFR SLK stock

Adding the SLK stock version of the M4A1 MFR in its various colors and barrel lengths

Alien314 committed:
Adding more default camos to the arsenal and new MCC stock for m4

adding the woodland and various alternative camos for default NATO clothes to the arsenal for Jubi

Adding new MCC stock to the arsenal.

Alien314 committed:
fix tap

Alien314 committed:
adjust animate carry to use ace properly?

Alien314 committed:
fix unit handling

Alien314 committed:
LightsOut/EMP module

Alien314 committed:
new maps

Alien314 committed:
RATS prefix and map names

blah2355 committed:
Add RATS server connection spotlight button to main menu (WIP)

Alien314 committed:
Maybe fixes / Use flags insignia

CfgFunctions will automatically tag and use files in functions folder, `file` property is relative to the prefix path.

Alien314 committed:
Real fixes

MomoTempo committed:
Adding a number of uniforms to the arsenal

Adding a large assortment of uniforms to the arsenal.

MomoTempo committed:
Adding Adv Medical items to Helo and Air Crews

MomoTempo committed:
Adding the new NVGs for Merc and fixing my ; for , issue on the uniforms

fixing ; substituted for commas throughout.

Added NVGs

Alien314 committed:
admins and jsrs vehicle sound curves

Alien314 committed:
Add RATS server connection spotlight button to main menu (WIP)

Removes vanilla "Spotlights" (middle square in the main menu) and adds one to quickly connect to the RATS server. Function connects to the server and autofills the password if it's saved in CBA's server password file.

Currently having config issues with finding the .paa file and function.

Alien314 committed:
remove CUP WP config, ace does it now

Alien314 committed:
btr80s and leo1 jsrs, better curve

Alien314 committed:
Adding a Number of Uniforms to the Arsenal. Blood for Pilots. NVGs for Merc.

Adding a large assortment of uniforms to the arsenal.

Alien314 committed:
Helo too loud :(, but now louder at mid to long

Alien314 committed:
Prevent old JSRS req issues

Alien314 committed:
Disable Ignore Bleed

Alien314 committed:
admin menu module

MomoTempo committed:
Additional Hats, Helmets and Sight Color Options

Added some alternative colors to some sights (HAMR, Bravo, ERCO).

Added some more hats (boonies!!!!)

Added some additional helmet options.

Alien314 committed:
unquote macros, unify _hasContact checks

Alien314 committed:
unify _hasApex checks

Alien314 committed:
Additional Hats, Helmets and Sight Color Options

Added some alternative colors to some sights (HAMR, Bravo, ERCO).

Added some more hats (boonies!!!!)

Added some additional helmet options.

Alien314 committed:
disable animate carry speed, sling wep drops on death

Alien314 committed:
novogorsk lighting config fix

Alien314 committed:
animate update

MomoTempo committed:
removing cringe helmets added by mistake

removing cringe CUP helmets

Alien314 committed:
removing cringe helmets added by mistake

removing cringe CUP helmets

Alien314 committed:
Allow internal view on some vics

Alien314 committed:
How could Jubi make me notice this

Alien314 committed:
animate update

Alien314 committed:
vurtually destructable

Alien314 committed:
fix empty mag setting being global

Alien314 committed:
animate sling improve, fix multi killed handler

Alien314 committed:
tactical reloads

Alien314 committed:
fix ace_isBelt caching, improve tac reload

MomoTempo committed:
Various Arsenal Requests by Users

several more rifles
several more mgs, associated magazine
some more vests
soft crew helmets
more laser/light modules

Alien314 committed:
dzn_xpi menu for MCC las/lights

Alien314 committed:
DUI pronouns/custom tags setting

Alien314 committed:
Various Arsenal Requests by Users

several more rifles
several more mgs, associated magazine
some more vests
soft crew helmets
more laser/light modules

Alien314 committed:
missing tools from crews

Alien314 committed:
fix arsenal

Alien314 committed:
Fix missing semi-colon

Alien314 committed:
fix uk vz59 greenmag config

Alien314 committed:
hide insignias

Alien314 committed:
fix dupe bug, quick load empty weapon

Alien314 committed:
Fix/improve impact smokes

MomoTempo committed:
adding weapons team leader role

adding weapons team leader role

Alien314 committed:
adding weapons team leader role

adding weapons team leader role

Alien314 committed:
Weapons TL Loadout

Alien314 committed:
what needs work to have quicker empty load

Alien314 committed:
animate more slings update

Alien314 committed:
cool mmg optic

Alien314 committed:
cars similar loudening to helos

Alien314 committed:
animate/sling fixes

Alien314 committed:
ace rangecard minor refactor

Alien314 committed:
animate sling update

MomoTempo committed:
Bullying ARs

Removing the ElcanM145 from ARs.

MomoTempo committed:
Also Adding More Bipod Colors

Adding more bipod colors.

MomoTempo committed:
adding more bipod colors

adding more bipod colors

Alien314 committed:
planes fix, 556 Grot

Alien314 committed:
new server

Alien314 committed:
Bullying ARs by removing an x from their 4x

Removing the ElcanM145 from ARs.
More Bipod colors.

Alien314 committed:
loadout radios + set unit role goggnogg

Alien314 committed:
set ace engi for crews

Alien314 committed:
expand animate object list, return true

Alien314 committed:
Add animate sling handles to TMF respawns

Alien314 committed:
breach fired EH

Alien314 committed:
new modules

Alien314 committed:
allow uncon on disconnect AI

Alien314 committed:
fix 5.56 galil recoil being 7.62

MomoTempo committed:
Numerous Updates to Weapons and Headware

New Helmets
New SMGs
New Rifles
New ARs
New Berets
New Ammos

MomoTempo committed:
Fixing 6.5x39 ammo

Fixing 6.5x39 ammo

MomoTempo committed:
fixed

fixeded

Bennay94 committed:
Update fnc_arsenal.sqf

added
Uniforms
-RACS Urban Uniform

Weapons
-CZ 805 BREN A2 and 807 various colors 
-CZ Skorpion EVO 3
-XM8 SAW various colors
-MG36
-M27 IAR
-Galil ARM 7.62

Ammo
-7.62 tracer and none tracer for Galil

Accessories 
-Bipod for FAL

MomoTempo committed:
removing 6.5

removing 6.5

Alien314 committed:
5.8 red tracer mags

Alien314 committed:
5.8 red ammo, rocket configs

Alien314 committed:
turn off firing range clones

Alien314 committed:
spectator count

Bennay94 committed:
fixed

fixed forgot to actually add the galil its self

Alien314 committed:
Update config.cpp

Alien314 committed:
allow number go down too

Alien314 committed:
skip when

Alien314 committed:
don't enable AI if respawn enabled

Alien314 committed:
Numerous Updates to Weapons and Headware

New Helmets
New SMGs
New Rifles
New ARs, less MMGs
New Berets
New Ammos

Alien314 committed:
Merge branch 'master' into patch-1

Alien314 committed:
ARs, general guns, and RACS urban uniform

added
Uniforms
-RACS Urban Uniform

Weapons
-CZ 805 BREN A2 and 807 various colors
-CZ Skorpion EVO 3
-XM8 SAW various colors
-MG36
-M27 IAR
-Galil ARM 7.62

Ammo
-7.62 tracer and none tracer for Galil

Accessories
-Bipod for FAL

Alien314 committed:
evo CBJ

Alien314 committed:
corrections

Alien314 committed:
M72A10

Alien314 committed:
M72A10, add 40mm HEDP penetrator

Alien314 committed:
engi 148 fix, aa/hat no bino fix, no wpn 148 if big team

Alien314 committed:
fix >:(

Alien314 committed:
new template, sorting

MomoTempo committed:
adding JCA respirators

Adding JCA respirators to _itemFacewear

Alien314 committed:
adding JCA respirators

Adding JCA respirators to _itemFacewear

Alien314 committed:
headers

Alien314 committed:
remove 2,1 missions

Alien314 committed:
fix 100rnd tracer, add JCA APRs to cbrn

MomoTempo committed:
Various Additions and Subtractions

removed some 7.62 bren rifles

added m110 in various colors stocks to sharpshooter
added l129a1 to sharpshooter
added base game CSAT DMR to sharpshooter
added none/red/yellow tracer 5.56 beta c mags to AR ammo
added various CUP m249s to AR
added some more camo options for various weapons
added 100rnd beta c mags to AR weapons

MomoTempo committed:
fixing tracer mag in red rather than green

fixing tracer mag in red rather than green

Alien314 committed:
Various Additions and Subtractions



removed some 7.62 bren rifles with 30 rounders from rifles.
removed 7.62 galil with 25 rounders from rifles.

added m110 in various colors stocks to sharpshooter added l129a1 to sharpshooter
added base game CSAT DMR to sharpshooter
added none/red/yellow tracer 5.56 beta c mags to AR ammo added various CUP m249s to AR
added some more camo options for various weapons
added 100rnd beta c mags to AR weapons

Alien314 committed:
reduce mass on used

Alien314 committed:
fix

Alien314 committed:
fix maaws 55 inheritance woopsy

Alien314 committed:
add acre objects to delete on disconnect, meme

Alien314 committed:
p90 greenmag box

Alien314 committed:
nerf QBZ mag weight

Alien314 committed:
fix QBZ 100Rnd tracer parent

Alien314 committed:
Prevent replacing disposable ammo in loadout

Alien314 committed:
fix show out of zeus

Alien314 committed:
ignore person weight

Alien314 committed:
remove forgotten debug message

Bennay94 committed:
M27 IAR ammo fix 

Added M27 IAR 100rnd ammo mags in red and yellow tracer to keep it competitive with the MG36 and XM8

removed M249 that could not take any optics

MomoTempo committed:
Removing 3 Cable Ties; Adding IR Strobe

Infinite cable ties means we don't need 4 in our uniform by default.

The size of 3 cable ties is approximately the same load as a single IR strobe which is also reusable, now. Rather than say "everyone bring an IR strobe" we can just automatically give them one that they could remove if they wanted and current loads don't really change otherwise.

Alien314 committed:
Removing 3 Cable Ties; Adding IR Strobe


Infinite cable ties means we don't need 4 in our uniform by default.

The size of 3 cable ties is approximately the same load as a single IR strobe which is also reusable, now. Rather than say "everyone bring an IR strobe" we can just automatically give them one that they could remove if they wanted and current loads don't really change otherwise.

Alien314 committed:
unlist other ties/strobes

Alien314 committed:
Merge branch 'master' of https://github.com/Alien314/mjb-arma

Alien314 committed:
M27 IAR ammo fix, remove 249 para

Added M27 IAR 100rnd ammo mags in red and yellow tracer to keep it competitive with the MG36 and XM8

removed M249 that could not take any optics

Alien314 committed:
arsenal active check

Alien314 committed:
fix _grp undefined in other roles

Alien314 committed:
add acre default radio to respawns

Alien314 committed:
fix sand mags

Alien314 committed:
disable mag collision with trees and walls

Alien314 committed:
hmmv DR adjust

Alien314 committed:
tacReload velocity and pos tweak

MomoTempo committed:
SCARs and NBCD suits

Adding NBCD suits to Uniforms

Adding various SCARs to Rifle, Grenade Launchers, and Sharpshooter.

MomoTempo committed:
Adding RShG2 to arsenal

Adding RShG2 to arsenal

MomoTempo committed:
adding purple yellow and orange impact smoke

adding purple yellow and orange impact smokes

Alien314 committed:
SCARs and NBCD suits, RPG-7 gone ;-;

Adding NBCD suits to Uniforms

Adding various SCARs to Rifle, Grenade Launchers, and Sharpshooter.

Removed RPG-7 from SF.

Alien314 committed:
JCA NBCD in CBRN toggle

Bennay94 committed:
added many hats and some ak stuff

Alien314 committed:
added many hats and some ak stuff

added many ECH and PASGT options

added some fancy AK stuff

added more LDF larp stuff

Alien314 committed:
some shotgun variants

Alien314 committed:
fix _unit undef

Alien314 committed:
return AR backpack belts

Alien314 committed:
crye tropentarn fix

Alien314 committed:
QBZ dispersion fix

MomoTempo committed:
Adding GL-less Rifle Respawn and SF Respawn

Adding Rifle Respawn and SF Respawn roles for reinforcements.

MomoTempo committed:
correcting names

correcting respawn role names

MomoTempo committed:
Player Loadouts for Respawns

Adding rifleRespawn classes (basically the same thing but without GL access)
Adding sfRespawn classes (basically the same thing but without GL access and is an SF RATS operator)

Alien314 committed:
Adding GL-less Rifle Respawn and SF Respawn

Adding Rifle Respawn and SF Respawn roles for reinforcements.

Alien314 committed:
Player Loadouts for Respawns

Adding rifleRespawn classes (basically the same thing but without GL access) Adding sfRespawn classes (basically the same thing but without GL access and is an SF RATS operator)

Alien314 committed:
toLower roles

Alien314 committed:
Light Blue and Pink smokes

Alien314 committed:
m72A10 adjust, lblue and pink smokes

Alien314 committed:
Light blue and pink smokes

Alien314 committed:
CUP SUV JSRS sounds and car boost

Alien314 committed:
RshG buff, allowInf for sure, TBG safety

Alien314 committed:
GL dispersion/distancebasedcooldown

Alien314 committed:
6rnd shells for entry

Alien314 committed:
Moved to config tweaks JSRS_noMedGrunt

Alien314 committed:
rng ai tweak

Alien314 committed:
we hates the sling mod

Alien314 committed:
GP-25 Tweaks fix

Alien314 committed:
VCOM/TCL Module

Alien314 committed:
VCOM/TCL Module

Alien314 committed:
muzzle speed/slot options

Alien314 committed:
Some optimization, allyChance CD trigger?

Alien314 committed:
More muzzle devices for stuff

Alien314 committed:
cTab update

Alien314 committed:
rush

MomoTempo committed:
Expanding _itemPackMedium to include all Carryall variants

Noticed that we don't have access to all the carryalls in _itemPackMedium. Now we do.

Alien314 committed:
Expanding _itemPackMedium to include all Carryall variants

Noticed that we don't have access to all the carryalls in _itemPackMedium. Now we do.

Removed "B_Carryall_cbr" it no longer seems to exist.

Alien314 committed:
return cbr from aceax bork

Alien314 committed:
lock, emp, and AIuncon module in 3den

Alien314 committed:
rng ai optimization pass, prevent apply to logic

Alien314 committed:
fix arg

Alien314 committed:
isEqualTo

Alien314 committed:
No other gun (ump)

Alien314 committed:
fix ump yellow

Alien314 committed:
More muzzle devices

Alien314 committed:
SF silencer/pistol mag fix

Alien314 committed:
JCA ump speedloadable and 50m min zero

Alien314 committed:
fix vehicle bft

Alien314 committed:
pride and trans bands, canada less white

Alien314 committed:
45 smg 50m zeros

MomoTempo committed:
Various Arsenal Updates AKs, Scopes, GLs, 4x Optic for MMG, random uniform

Added various AK pattern weapons as options.
Grenade Launcher variants added as well.

4x Scope for the MMG

random tanker uniform for tankers.

Alien314 committed:
Various Arsenal Updates AKs, Scopes, GLs, 4x Optic for MMG, random un…

Fish gun XM8

Added various AK pattern weapons as options.
Grenade Launcher variants added as well.

4x Scope for the MMG

random tanker uniform for tankers.

Alien314 committed:
set role without arsenal, remeber ticked

Alien314 committed:
acre scale and screm settings

Alien314 committed:
update cover oops

Alien314 committed:
group react delay, move bino swap, optimization

Alien314 committed:
prevent scripted fire at non-hostile, optimize,

Alien314 committed:
module attributes

Bennay94 committed:
many many BDU's

added a ton of BDU variants and some other assorted uniforms

added JCA optics as per momo

Alien314 committed:
comma

MomoTempo committed:
Removing 6.8 GLs.

Removing the 6.8 GL options.

Alien314 committed:
acre backup shenanigans

Alien314 committed:
allow shooting empty

Alien314 committed:
disable aceax version check

Alien314 committed:
no TI ENVGs

Alien314 committed:
loot fix the try once more

Alien314 committed:
return of goofy ragdolls probably

Alien314 committed:
spectate tv

Alien314 committed:
suppressor arsenal box

Alien314 committed:
+dist, base support box

MomoTempo committed:
Removing Higher Magnification scopes from MMG

We added them and they're just too good. For now, I'm suggesting we scale this back. The 6x is definitely too much.

Alien314 committed:
Cookoff Grace and Air Damage

Alien314 committed:
Cookoff grace

Alien314 committed:
missing semi colon

Alien314 committed:
acre shift and amphibious speech

Alien314 committed:
many many BDU's

added a ton of BDU variants and some other assorted uniforms

added JCA optics as per momo

Alien314 committed:
Removing 6.8 GLs.

Removing the 6.8 GL options.

Alien314 committed:
Removing Higher Magnification scopes from MMG

We added them and they're just too good. For now, I'm suggesting we scale this back. The 6x is definitely too much.

Alien314 committed:
no TI envgs, missing hand smokes for leads

Alien314 committed:
g18

Alien314 committed:
fix sim disable

Alien314 committed:
fix condition

MomoTempo committed:
Adding Mk 16 SCAR (5.56) to rifles

Adding Mk 16 SCAR (5.56) to rifles

Bennay94 committed:
helmets

fixed pasgt extended arsenal issue from my laziness 

added some other assorted helmets to match existing camo patterns

Bennay94 committed:
helmets + grach pistol

MomoTempo committed:
Removing another 4x scope

MomoTempo committed:
Removing 4x from MMG default loadout

Removing 4x from MMG default loadout, too strong. Sowwy.

Alien314 committed:
mag lost mags

Alien314 committed:
fix condition bump max setting

Alien314 committed:
exit when nil magazine

Alien314 committed:
multi plate regen fix

Alien314 committed:
check isNull, air damage mid game disable

Alien314 committed:
am dumb, _directVolume exists

Alien314 committed:
adjust calcs to not reduce volume

Alien314 committed:
prevent mag loss, remove greenmag empty

MomoTempo committed:
Adding M17/Mags, L85 GL, NV Capable MAAWS

Adding M17/Mags - User Request

L85 GL version typo fixed?

NV Capable MAAWS for night operations and its ammo.

MomoTempo committed:
Adding 2 point sling to inventory

Adding 2 point sling to inventory

MomoTempo committed:
Adding Slings to everyone's arsenal

Everyone gets access to the 2 point sling.

Alien314 committed:
Adding Mk 16 SCAR (5.56) to rifles, remove more MMG 4xs

Adding Mk 16 SCAR (5.56) to rifles
remove more MMG 4xs

Alien314 committed:
helmets + grach

fixed pasgt extended arsenal issue from my laziness

added some other assorted helmets to match existing camo patterns

add grach

Alien314 committed:
Removing 4x from MMG default loadout. Adding Sling to everyone.

Removing 4x from MMG default loadout, too strong. Sowwy.

Adding 2 point sling to default inventory.

Alien314 committed:
Adding M17/Mags, L85 GL, NV Capable MAAWS

Adding M17/Mags - User Request

L85 GL version typo fixed?

NV Capable MAAWS for night operations and its ammo.

Alien314 committed:
Adding Sling to everyone's arsenal

Everyone gets access to the 2 point sling.

Alien314 committed:
plates feature integration prep

Alien314 committed:
yeet old 2,2 templates

Alien314 committed:
fix infinite grenades

MomoTempo committed:
Adding Various PLA items to the arsenal

Adding Various PLA items to the arsenal

Weapons
Light Backpacks
Helmets
Uniforms
Vests

Alien314 committed:
Adding Various PLA items to the arsenal

Adding Various PLA items to the arsenal

Weapons
Light Backpacks
Helmets
Uniforms
Vests

Pistol magazine for pistol.

Alien314 committed:
fix applies to local AI

Alien314 committed:
fix inf ammo when no gesture

Alien314 committed:
Merge branch 'master' of https://github.com/Alien314/mjb-arma

Alien314 committed:
delay ctab drawing on map

Alien314 committed:
Not and draw player on map/player color

Alien314 committed:
5.7 buff

Alien314 committed:
5.8x21

Alien314 committed:
shell count loading

Alien314 committed:
vanilla 5.8 mag well

Alien314 committed:
missing surface effects

Alien314 committed:
fix single TL-2 pylon desc

Alien314 committed:
move pla contact vests and gls, 30rnd greenmag

Alien314 committed:
increase grenade impact volume

Alien314 committed:
softer soft

MomoTempo committed:
MCC update

New:
m4s
mk 18s
aimpoints
spuhr mounts

MomoTempo committed:
Adding default equipment to TankCrew

Giving default Equipment items to TankCrew.

Alien314 committed:
Tracers note

Alien314 committed:
prevent non player ppfx, destroy effect on death

Alien314 committed:
prevent cringe aceax shenanigan

Alien314 committed:
clutter cutter by Raimo

Alien314 committed:
Echo disable

Alien314 committed:
default air damage off, for VPS

Alien314 committed:
MCC update and additional TankCrew Equipment

New:
m4s
mk 18s
aimpoints
spuhr mounts

Alien314 committed:
ghillies and nvg glasses

Alien314 committed:
new MCC sight zeroing locks

Alien314 committed:
disable captive roll

Alien314 committed:
pronoun color and porpl

Alien314 committed:
improve clutter cut

Alien314 committed:
prevent turret crew FFV allowing driver dismount

Alien314 committed:
disable captive cTab markers

Alien314 committed:
make MCC lasers consistent

MomoTempo committed:
Removing Golf's MAT and MMG access to rangefinders, laser designators and binoculars.

Removing Golf's MAT and MMG access to rangefinders, laser designators and binoculars.

Alien314 committed:
awawawawa

Alien314 committed:
uav ai in editor, heli maxFordingDepth

Alien314 committed:
fix qsz92 greenmag

Alien314 committed:
slingloadable supply boxes

Alien314 committed:
less maaws duping

Alien314 committed:
no sutures

Alien314 committed:
posawa

Alien314 committed:
Lessen Golf gunners' ability to self-range for teamwork

Removing Golf's MAT and MMG access to rangefinders, laser designators and binoculars.

Alien314 committed:
special team teamwork change

Alien314 committed:
MTP-Winter

Alien314 committed:
m72uning, disposable bunker busters

Alien314 committed:
muzzle velocity needs thrust changes

Alien314 committed:
HEDP/MDP fixes

Alien314 committed:
allow manual IP change in future mebbe

Alien314 committed:
remove ranging in MMGAmmo

Alien314 committed:
port too, why not

Alien314 committed:
zeus assignation failsafe

Alien314 committed:
owner is wiped before postInit

Bennay94 committed:
ir tracers

added IR tracers for 5.56 and 7.62x51 for use during night missions

Alien314 committed:
tacReload mag collision fix attempt 2

Alien314 committed:
fix init/adjust purple team color

Alien314 committed:
smaller door lock radius in zeus

Alien314 committed:
run animate init for respawns

Alien314 committed:
allow zeus to use tmf admin stuff

Alien314 committed:
failsafe for stuck map anim in binos

Alien314 committed:
ir tracers

added IR tracers for 5.56 and 7.62x51 for use during night missions

Alien314 committed:
PLA audibility

Alien314 committed:
fix dumb loop, prevent client stuff on server

Alien314 committed:
IR Tracer for (most) NATO ARs

Alien314 committed:
Move CBJ, IR Tracer scale

Alien314 committed:
IR Tracer, Apex Minimi mag fix, AH64 mags

Alien314 committed:
fix RC distance volume, remove acreShift

Alien314 committed:
AH64 mags

Alien314 committed:
AH64 mags, WP pilot and SP helmets

Alien314 committed:
fix 32 ACP greenmag class name

Alien314 committed:
fix sa vz 61 acc

Alien314 committed:
pilot WP helmets

Alien314 committed:
ace nv integrated nvg tweak

Alien314 committed:
21rnd M17 mag

Alien314 committed:
JSRS complete no kick

MomoTempo committed:
Updating MMG and MAT to not have leader equipment

In the case where WpnTL is absent this should address MAT and MMG from getting leader equipment.

Alien314 committed:
Updating MMG and MAT to not have leader equipment

In the case where WpnTL is absent this should address MAT and MMG from getting leader equipment.

Alien314 committed:
bai needs less fix, with suppress event

Alien314 committed:
prevent server break from better inventory

Alien314 committed:
standalone ver

Alien314 committed:
fix bad else endif

Alien314 committed:
T72 engine protection

Alien314 committed:
fix turretless condition

Alien314 committed:
fix inheritance on mcc vis combo

Alien314 committed:
galaxy armband for Euchre

MomoTempo committed:
New MCC Weapons and Sights

New MCC Weapons and Sights.

New 5.56 weapons.

optics added:
1x
3x

MomoTempo committed:
Additional Change to Add Suppressors to SF Respawn

Adding Suppressors to SF Respawn

MomoTempo committed:
Adjusting SF MAT AT Options

Adjusting SF MAT's AT options.

MomoTempo committed:
Updating SF MAT Ammo

Updating SF MAT Ammo

MomoTempo committed:
Updating Playerloadout

Fixing SFMAT starting weapon.

MomoTempo committed:
Adding the Scrim Helmet to itemSF

Adding the Scrim Helmet to itemSF

Alien314 committed:
MCC dot ranging and new sights

Alien314 committed:
m106 fog

Alien314 committed:
fix ace effects applying/slot change

Alien314 committed:
Updating Playerloadout, SF MAT's starting launcher.

Fixing SFMAT starting launcher.

Alien314 committed:
yeet zeus kb versioning

Alien314 committed:
Merge branch 'master' of https://github.com/Alien314/mjb-arma

Alien314 committed:
New MCC Weapons and Sights, SF Respawn Tweaks, SF MAT Tweaks

New MCC Weapons and Sights.

New 5.56 weapons.

optics added:
1x
3x

Alien314 committed:
Merge branch 'master' of https://github.com/Alien314/mjb-arma

Alien314 committed:
fog and r20 rahe

Alien314 committed:
no sting for sf mat

MomoTempo committed:
Removing problematic scope

Didn't notice the Romeo8t is a 5x scope. Removing this flipper.

MomoTempo committed:
Adding RATS Breacher and SF Breacher. Fixing Leader Check

RATS Breacher class creation

MAT/MMG check is now ">= 3" rather than "> 3"

MomoTempo committed:
Adding Breacher and SFBreacher classes

Adding starting arsenal for Breacher and SF Breacher.

MomoTempo committed:
Adding Alien's Rangefinder Remover

Adding Rangefinder Remover from Alien that should address MAT and MMG from getting rangefinders incorrectly.

Alien314 committed:
Removing problematic scope, Add Breacher roles

Didn't notice the Romeo8t is a 5x scope. Removing this flipper.
Add breacher roles

Alien314 committed:
Adding Breacher and SFBreacher classes

Adding starting arsenal for Breacher and SF Breacher.

Alien314 committed:
I got function wrong

Alien314 committed:
forgotten =

Alien314 committed:
fix breacher loadout explosives

Alien314 committed:
noMagnified module, improve noTab

Alien314 committed:
m203 249 optics

Alien314 committed:
opfor switchblade, crew fix for ind

Alien314 committed:
mini yabhon noise

Alien314 committed:
fix mcc laserpos

MomoTempo committed:
Enabling the Red Cross Armband

Alien314 committed:
prevent gesture fix on sling swap

Alien314 committed:
script debug, novoice disconnects

Alien314 committed:
stop a3ti handling every key

Alien314 committed:
specifically slings actually

Alien314 committed:
bit more distance for 600

MomoTempo committed:
Adding Various Weapons and Upcoming Ghillies for SF and others

* Added OP weapons in the follow categories
_itemSFWeaponGL
_itemSFWeaponMG
_itemSFWeaponRifle
_itemSFWeaponDMR

* Added _itemSFScoutGhillies for sniper/spotter and SF users.

MomoTempo committed:
Removing Demo from SF. Adding Slings to Breachers.

* Removing Demo from all SF. Breachers and Leaders still have access.
* Adding Slings to Breachers.

MomoTempo committed:
Fixing Accidental Sling Removal

MomoTempo committed:
Should fix my fix breaking everything

I hate github

Alien314 committed:
Enabling the Red Cross Armband

Enabling the Red Cross Armband

Alien314 committed:
Adding Various Weapons and Upcoming Ghillies for SF and others

  Added OP weapons in the follow categories:
    _itemSFWeaponGL
    _itemSFWeaponMG
    _itemSFWeaponRifle
    _itemSFWeaponDMR

    Added _itemSFScoutGhillies for sniper/spotter and SF users.

Alien314 committed:
syntax, glass cross, removed unit gear in template

Alien314 committed:
fix undef

Alien314 committed:
Fix RHS errors
